limits-and-troubleshooting/modification-in-resizing-method-for-ABS-representations.md

```diff
@@ -0,0 +1,34 @@
+# Modifications in resizing method for ABS representations
+
+So far, we resized the ABS representations using [pad_resize crop mode](../features/video-transformation/resize-crop-and-other-common-video-transformations.md#pad-resize-crop-strategy---cm-pad_resize), to a strict aspect ratio of 16:9 for all the resolutions. This added black color padding if the input didn’t have an aspect ratio of 16:9.
```

Comment:
- Specific dates instead of "so far"
- Replace "we" language with "ImageKit" or "you" (as in user, wherever applicable) language
- Present tense for current behavior, until it changes